docker/dockerfiles/Dockerfile.worker

```diff
@@ -11,6 +11,7 @@ FROM ubuntu:16.04
 MAINTAINER CodaLab Worksheets <codalab.worksheets@gmail.com>
 
 ENV DEBIAN_FRONTEND noninteractive
+ENV PATH="/bin/bash:${PATH}"
```

Comment:
Why is this necessary, especially since you have `/bin/bash` in the docker command above?

Reply:
Theoretically yes. However, the existing `bash` is already linked to `/bin/bash`, but the program still can't find it when executing. I did several ways of testing, they are documented at [here](https://github.com/codalab/codalab-worksheets/issues/1704#issuecomment-575926090). It appears setting only the `PATH` in the environment variable still can't help the program to navigate to the right executable. So I feel the safest way is to add BASH executable directly to `PATH` and call `/bin/bash` in the code.